README.md

```diff
@@ -4,6 +4,9 @@ project description:
 make a thing that'll cycle through 3 images at a time. when you click on one it'll increase vote count and produce 3 images, none of which are supposed to have been in the group you were just previously viewing. 
 
 
+3/19:
+Results list are spaced out from each other and the edge of the screen, and will be displayed after the user has gone through 25 iterations, at which point the image rotations will stop. Counter is now improved and should function properly
+
 3/19 Updates:
 Got scaffolding set up, images cycling with no repeats then displaying results afterwards. got some help with Justin & Jason, and lots of advice from TAs. Worked with Jason & Stephan on for the wireframe.html which I only just afterwards realized should just be implemented into the index.html. Got to wireframe portion at the of the lab, having trouble merging it into index. Most of it should be carried over, but not all is displaying as I'd like. 
 
```

Comment:
As noted before, your README needs to be user instructions (teach me how to use the app) not your log entries